app/src/main/java/org/fossasia/openevent/app/adapters/EventListAdapter.java

```diff
@@ -0,0 +1,75 @@
+package org.fossasia.openevent.app.adapters;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+import org.fossasia.openevent.app.R;
+import org.fossasia.openevent.app.model.Event;
+import org.fossasia.openevent.app.views.EventDetailsActivity;
+
+import java.util.List;
+
+
+public class EventListAdapter extends RecyclerView.Adapter<EventListAdapter.EventRecyclerViewHolder>{
+
+    private List<Event> events;
+    private Context context;
+
+    public EventListAdapter(List<Event> events, Context context) {
+        this.events = events;
+        this.context = context;
+    }
+
+    @Override
+    public EventRecyclerViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View itemView = LayoutInflater.from(parent.getContext()).inflate(R.layout.event_layout, parent, false);
+
+        return new EventRecyclerViewHolder(itemView);
+    }
+
+    @Override
+    public void onBindViewHolder(final EventRecyclerViewHolder holder, int position) {
+
+        final Event thisEvent = events.get(position);
+        holder.eventTitle.setText(thisEvent.getName());
+        Picasso.with(context).load(thisEvent.getThumbnail()).into(holder.eventImage);
```

Comment:
This PR is actually just fixing lint errors and naming, if we continue developing on this PR, we won't move forward. Our motive should be to make this PR mergable so there are no crashes and then start fixing bugs in other issues